### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by 6543**
Update Dependencies with go 1.15.2

### CODE REVIEW

**Comment by 6543** [comment]
@rexim do you have time to take a look at this dependency upgrades :)

If I have missed something just tel me

File: go.mod [hunk 1]
```diff
@@ -4,6 +4,8 @@ go 1.13
 
 require (
 	github.com/pkg/errors v0.9.1
-	gopkg.in/go-ini/ini.v1 v1.52.0
-	gopkg.in/yaml.v2 v2.2.8
+	github.com/smartystreets/goconvey v1.6.4 // indirect
```
Comment (rexim) [inline go.mod:7]: What is this package and why are we indirectly depend on it? The description says "About Go testing in the browser." why do we need testing in the browser for a text command line utility? Is it possible to not depend on that?

**Review by rexim — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (6543) [reply go.mod:7]: let me check the libs ...

Comment (6543) [reply go.mod:7]: `go mod tidy` added this, I normaly use it to shrink go.sum file ... in this case It have added this dependencd

**Comment by 6543** [comment]
@rexim done :)

**Comment by 6543** [comment]
@rexim any updates on this?

**Comment by rexim** [comment]
@6543 looks good to me! :+1: Thank you for your contribution!

**Review by rexim — APPROVED** [review]

**Pull request merged**